### PR TITLE
Rename client for workshop query. Add javadoc for WorkshopQueryResponse.

### PR DESCRIPTION
**Pull request by Aquerr**

### CODE REVIEW

File: src/main/java/io/github/aquerr/steamwebapiclient/response/WorkShopQueryResponse.java [hunk 1]
```diff
@@ -11,117 +15,342 @@
...
+         */
         @JsonProperty("next_cursor")
         private String nextCursor;
+
+        /**
+         * The list of found workshop items.
+         */
         @JsonProperty("publishedfiledetails")
-        private List<PublishedFileDetails> publishedFileDetails;
+        private List<PublishedFileDetails> publishedFileDetails = new ArrayList<>();
```
Comment (Aquerr) [inline src/main/java/io/github/aquerr/steamwebapiclient/response/WorkShopQueryResponse.java:37]: It should be like that. We should always return an empty list if we do not find any results.

File: src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java [hunk 2]
```diff
@@ -70,18 +70,18 @@ public SteamWebApiClient(String apiKey, HttpClient httpClient, ObjectMapper obje
...
     }
 
     /**
-     * Gets the {@link SteamWorkshopWebApiClient}
-     * @return the {@link SteamWorkshopWebApiClient}
+     * Gets the {@link SteamPublishedFileWebApiClient}
+     * @return the {@link SteamPublishedFileWebApiClient}
      */
-    public SteamWorkshopWebApiClient getWorkshopWebApiClient()
+    public SteamPublishedFileWebApiClient getWorkshopWebApiClient()
```
Comment (mateo9x) [inline src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java:84]: Do you change also getter name or it should stay as it is?

**Review by mateo9x — APPROVED** [review]

File: src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java (hunk 2 above)
Comment (Aquerr) [reply src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java:84]: Ah, right. I will change it.

Comment (mateo9x) [reply src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java:84]: > Owner Author

Just askin' :P

**Pull request merged**